README.md

```diff
@@ -10,6 +10,12 @@ Background job development in a breeze.
 # Installation
 Easy to install it via NuGet:
 
+- **Azure Active Directory background jobs**
+
+```shell
+PM > Install-Package Arcus.BackgroundJobs.AzureActiveDirectory
+```
+
```

Comment:
Good that this is added to this overview, so it's kept up to date.
However, I wonder if it would be helpfull if we add a short description per job (1 / 2 lines) that explains what you can do with it ?
I'll add a discussion on this topic, so this PR can be closed.